src/sercon.c

```diff
@@ -99,6 +99,12 @@ static void sercon_putchar(u8 chr)
     }
 }
 
+static void sercon_term_reset(void)
```

Comment:
@miczyg1 one line documentation what we do in this function would be great.

Reply:
Done.